src/common/Pagination/Pagination.tsx

```diff
@@ -135,15 +150,26 @@ export const Pagination: FC<PaginationProps> = forwardRef<any, PaginationProps>(
 });
 
 const NavigationButton: FC<{
-  onClick: (e: React.MouseEvent<HTMLDivElement>) => void,
+  onClick: () => void,
   mod: string[],
   disabled?: boolean,
+  hotkey?: string,
 }> = (props) => {
```

Comment:
can we change props to `{ onClick, mod, disabled, hotkey }` and reference the prop values accordingly? just to be more consistent with the rest of our code